samples/sensor/mma8451q/sample.yaml

```diff
@@ -0,0 +1,2 @@
+sample:
+  name: MMA8451Q Accelerometer/Magnetometer Sensor
```

Comment:
Need to add:
```
tests:
  test:
    harness: sensor
    platform_whitelist: frdm_kl25z
    tags: samples sensor                                                        
    depends_on: i2c
```
You can verify by running: `$ sanitycheck -T samples/sensor/mma8451q/ -l`

Before the change, you'll see:
```
Cleaning output directory /home/maureen/zephyr/sanity-out
Selecting all possible platforms per test case
Building testcase defconfigs...
0 tests selected, 0 tests discarded due to filters

0 of 0 tests passed with 0 warnings in 1 seconds
```

and after the change:
```
Cleaning output directory /home/maureen/zephyr/sanity-out
Selecting all possible platforms per test case
Building testcase defconfigs...
1 tests selected, 121 tests discarded due to filters
total complete:    1/   1  100%  failed:    0
1 of 1 tests passed with 0 warnings in 16 seconds
```

Reply:
I added it - it went green on my laptop.  Let's see with the CI